README.md

```diff
@@ -14,99 +15,186 @@ Keep reading to get started or check out the [Roadmap](#roadmap) to see the feat
+Setting up your database
 
-Once you've added your `DATABASE_URL` to your `.env` file, you're ready to migrate your database:
+#### Step 4.1 - setting the database on macOS
 
-```
-yarn rw prisma migrate dev
-```
+The Stripe Superstore uses Postgres, so if you do not have it setup locally, the following paragraphs,
+inspired by this YouTube presentation [How to Install PostgreSQL and PgAdmin 4 on Mac OS](https://www.youtube.com/watch?v=u3w2Rrp9HIQ) are for you.
 
-Now you should be able to start the dev server!
+1: Point the browser to https://www.postgresql.org/.
 
-```
-yarn rw dev
+2: Click on the download link.
+
+3: Click on macOS link on the download page
+
+4: Click on [download the installer](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads) link to get EDB certified installer.
+
+5: On the page [Download PostgreSQL](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads) pick the version 14.3 for mac OSX.
+
+6: Follow the [installation instructions](https://www.enterprisedb.com/postgres-tutorials/installation-postgresql-mac-os).
+
+7: run the downloaded postgresql-14.3-1-osx.dmg package which will start EDB PostgreSQL Setup Wizard. Take all default settings that are proposed by the wizard.
+
+8: Once the Wizard finishes, you can verify the installation of all component by issuing the command `ps -ef | grep postgres`. You should see:
+
+```zsh
+nik$ ps -ef | grep postgres
+  504 48772 48771   0  6:54PM ??         0:00.00 postgres: logger
+  504 48774 48771   0  6:54PM ??         0:00.01 postgres: checkpointer
+  504 48775 48771   0  6:54PM ??         0:00.02 postgres: background writer
+  504 48776 48771   0  6:54PM ??         0:00.02 postgres: walwriter
+  504 48777 48771   0  6:54PM ??         0:00.03 postgres: autovacuum launcher
+  504 48778 48771   0  6:54PM ??         0:00.16 postgres: stats collector
+  504 48779 48771   0  6:54PM ??         0:00.00 postgres: logical replication launcher
+  501 49879  2395   0  8:26PM ttys002    0:00.00 grep postgres
 ```
 
-5. Listening for webhooks
+___
+
+#### Step 4.2 - setting the database on Windows
+
+TBD
+___
+
+Once you've added your `DATABASE_URL` to your `.env` file, you're ready to migrate your database:
+
+`yarn rw prisma migrate dev`
+___
+
+### Step 5
+
+Listening for webhooks
 
 In tandem with the dev server, you'll want to use the stripe CLI to start a process that listens for webhooks:
 
-```
-stripe listen --forward-to 'localhost:8911/stripeWebhooks'
-```
+`stripe listen --forward-to localhost:8911/stripeWebhooks`
 
 Make sure to pass the serverless function that's going to receive webhooks to the `--forward-to` flag.
+In this app this serverless function's URL is <localhost:8911/stripeWebhooks> and the source is at `/Users/nik/dev/work/redwood/stripe/rw-integration/example-store-stripe/api/src/functions/stripeWebhooks`.
```

Comment:
I agree with making the link between `localhost:8911/stripeWebhooks` and the serverless function more obvious, but the first half reads a little too repetitive (the URL is in the code block above) and the second half, too specific (since it's your machine).

Reply:
I agree wholeheartedly 😸